apps/src/componentLibrary/segmentedButtons/SegmentedButtons.story.tsx

```diff
@@ -200,12 +200,12 @@ TypesOfSegmentedButtons.args = {
         {
           label: 'Label Two',
           value: 'label-two-il',
-          iconLeft: {iconName: 'check', iconStyle: 'solid', title: 'check'},
+          iconLeft: {iconName: 'house', iconStyle: 'solid', title: 'check'},
```

Comment:
Thanks for rendering different types of icons here! I think we can merge as-is, but maybe we can update `title` of these in another PR.

Reply:
Oh yeah, thanks a lot for spotting these, I forgot about them. Will update in a followup PR